concent_api/core/tests/test_unit_validation.py

```diff
@@ -268,3 +271,50 @@ def test_that_if_frames_are_not_one_after_the_other_method_should_pass(self):
     def test_that_if_frames_are_not_integers_method_should_raise_exception(self):
         with self.assertRaises(FrameNumberValidationError):
             validate_frames(['1', '2'])
+
+
+class TestValidateComputeTaskDef(object):
+    compute_task_def = None
```

Comment:
I think there is no need to define it here.

Reply:
it is ;)
either this or:
`W:283, 8: Attribute 'compute_task_def' defined outside __init__ (attribute-defined-outside-init)`

Reply:
Right, because you don't inherit from `TestCase`. It is ok then.